tests/sentry/interfaces/test_stacktrace.py

```diff
@@ -639,6 +639,25 @@ def test_collapse_recursion(self):
             'io.sentry.example.Application', 'throwError'
         ])
 
+    def test_frame_hard_limit(self):
+        interface = Stacktrace.to_python(
+            {
+                'frames': [
+                    {
+                        'filename': 'Application.java',
+                        'function': 'main',
+                        'lineno': i,
+                    } for i in range(1, 252)
```

Comment:
hardcoded to be equal to the current limit from settings, which is not the best

Reply:
Can we just pull in `settings.SENTRY_STACKTRACE_FRAMES_HARD_LIMIT` and add some number, that's a little better at least